s5_base/run.sh

```diff
@@ -125,62 +140,107 @@ fi
 
 # train a monophone system
 if [ $stage -le 7 ]; then
-
+    
     echo "Train monophone system"
-    steps/train_mono.sh --nj $nj_train --cmd "$train_cmd --mem 4G" --boost-silence 1.25 \
-      data/train_5k data/lang exp/mono
-
-    echo "Mono alignment. Align train_5k to mono"
-    steps/align_si.sh --nj $nj_train --cmd "$train_cmd" data/train_5k data/lang \
-      exp/mono exp/mono_ali_5k
-
-    echo "First triphone on train_5k, delta + delta-delta features"
-    steps/train_deltas.sh --cmd "$train_cmd --mem 4G" 2000 10000 data/train_5k \
-      data/lang exp/mono_ali_5k exp/tri1
+    steps/train_mono.sh \
+    --nj $nj_train \
+    --cmd "$train_cmd --mem 4G" \
+    --boost-silence 1.25 \
+    data/train_5k data/lang exp/mono
+    
+    echo "Mono alignment. Align train_10k to mono"
+    steps/align_si.sh \
+    --nj $nj_train --cmd "$train_cmd" \
+    data/train_10k data/lang \
+    exp/mono exp/mono_ali_10k
+    
+    echo "First triphone on train_10k, delta + delta-delta features"
+    steps/train_deltas.sh \
+    --cmd "$train_cmd --mem 4G" \
+    2000 10000 \
+    data/train_10k data/lang \
+    exp/mono_ali_10k exp/tri1
 fi
 
 # Train LDA + MLLT
 if [ $stage -le 8 ]; then
-    echo "Aligning the train_10 set to tri2"
-    steps/align_si.sh --nj $nj_train --cmd "$train_cmd" data/train_10k data/lang exp/tri1 \
-      exp/tri1_ali_10k
-
+    echo "Aligning the train_10 set to tri1"
+    steps/align_si.sh \
+    --nj $nj_train \
+    --cmd "$train_cmd" \
+    data/train_10k data/lang \
+    exp/tri1 exp/tri1_ali_10k
+    
     echo "Train LDA + MLLT"
-    steps/train_lda_mllt.sh --cmd "$train_cmd --mem 4G"  4000 40000 data/train_10k \
-      data/lang exp/tri1_ali_10k exp/tri2
+    steps/train_lda_mllt.sh \
+    --cmd "$train_cmd --mem 4G"  \
+    4000 40000 data/train_10k \
+    data/lang exp/tri1_ali_10k exp/tri2
 fi
 
 # Train LDA + MLLT + SAT
 if [ $stage -le 9 ]; then
     echo "Aligning the full training set to tri2"
-    steps/align_si.sh --nj $nj_train --cmd "$train_cmd" data/train data/lang exp/tri2 \
-      exp/tri2_ali
-
+    steps/align_fmllr.sh \
+    --nj $nj_train \
+    --cmd "$train_cmd" \
+    data/train data/lang \
+    exp/tri2 exp/tri2_ali
+    
     echo "Train LDA + MLLT + SAT"
     steps/train_sat.sh --cmd "$train_cmd --mem 4G"  4000 40000 data/train \
-      data/lang exp/tri2_ali exp/tri3
+    data/lang exp/tri2_ali exp/tri3
 fi
 
 # Tri3 decoding
 if [ $stage -le 10 ]; then
     echo "Triphone tri3 decoding"
-    $train_cmd --mem 4G exp/tri3/log/mkgraph.log utils/mkgraph.sh data/lang_3g exp/tri3 exp/tri3/graph
```

Comment:
I see it's different for almost each call. 

In my subword script, I have different 

cmd calls for different actions but I guess that is just preferance